patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1504,12 +1504,21 @@
 +        return isBurning(world, pos) ? net.minecraft.pathfinding.PathNodeType.DAMAGE_FIRE : null;
 +    }
 +
++    /**
++     * @param state The state
++     * @return true if the block is sticky block (slime blocks)
```

Comment:
This javadoc should explain that the method is used for pistons. "sticky" is not very descriptive without that context.

Reply:
Done.